specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md

```diff
@@ -0,0 +1,247 @@
+winrt::event_token batteryToken;
+winrt::event_token powerToken;
+
+void MyRegisterPowerManagerCallbacks()
+{
+    batteryToken = PowerManager::BatteryStatusChanged([](IInspectable sender, auto args)
+        { OnBatteryStateChange(sender, args); });
+    powerToken = PowerManager::PowerSupplyStatusChanged([](IInspectable sender, auto args)
+        { OnPowerStateChange(sender, args); });
+    //...etc
+}
+
+void MyUnregisterPowerManagerCallbacks()
+{
+    PowerManager::BatteryStatusChanged(batteryToken);
+    PowerManager::PowerSupplyStatusChanged(powerToken);
+    //...etc
+}
+```
+
+<br>
+
+## API Details
+### Existing APIs
+This set of APIs is a subset of the union of the existing WinRT PowerManager class and the existing PowerSettingRegisterNotification API. 
+
+The existing PowerSettingRegisterNotification API takes 4 parameters:
+
+|     PowerSettingRegisterNotification   parameter     |     Description                                                                                                                                                      |
+|------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|     LPCGUID   SettingGuid                            |     A   GUID that represents the specific power setting in which the app is   interested.                                                                            |
+|     DWORD   Flags                                    |     This   is either DEVICE_NOTIFY_SERVICE_HANDLE if the Recipient is a service, or DEVICE_NOTIFY_CALLBACK   if the Recipient is a callback function.                |
+|     HANDLE   Recipient                               |     A   handle to the recipient of the notifications.                                                                                                                |
+|     PHPOWERNOTIFY   RegistrationHandle               |     Out   parameter: a handle to the registration, which the app can later use to   unregister for notifications, via the PowerSettingUnregisterNotification API.    |
+
+Reunion AppLifecycle will not bring this function forward in its current form. Instead, AppLifecycle will bring the functionality forward using the PowerManager pattern for the additional PowerSettingRegisterNotification notifications:
+-	Instead of a single register function that takes a GUID, the new API will expose multiple new events, mapping to each of the GUIDs that are brought forward. Not all GUIDs will be brought forward, as some of them are already covered by PowerManager events.
+-	For Reunion v1, the target consumers of the new APIs are scoped to apps not services - this eliminates the need for the Flags parameter. 
+-	The new API will adopt the PowerManager pattern for registering and unregistering callbacks, which eliminates the need for the register/unregister handles.
```

Comment:
Please don't use tabs; just `- The new ...`

Reply:
Updated.